beagle_cli

```diff
@@ -175,6 +184,22 @@ def _check_is_authenticated(config):
 
 # List commands
 
+def _get_runs_command(arguments, config):
+    run_id = arguments.get('--run-id')
+    page_size = arguments.get('--page-size')
+    params = dict()
+    if page_size:
+        params['page_size'] = page_size
+    url = urljoin(BEAGLE_ENDPOINT, API['run'])
+    if run_id:
```

Comment:
Can you separate list and get calls? And when you do a get there will be no pagination, since you getting only one run object.